service/dps/storm/topologies/indexing/src/main/java/eu/europeana/cloud/service/dps/storm/topologies/indexing/bolts/IndexingBolt.java

```diff
@@ -98,30 +110,50 @@ private void init() throws IndexerConfigurationException, URISyntaxException {
             IndexingSettings indexingSettingsForPreviewDbInAnotherEnv = settingsGeneratorForAnotherEnv.generateForPreview();
             IndexingSettings indexingSettingsForPublishDbInAnotherEnv = settingsGeneratorForAnotherEnv.generateForPublish();
 
-            indexerFactoryForPreviewDbInDefaultEnv = new IndexerFactory(indexingSettingsForPreviewDbInDefaultEnv);
-            indexerFactoryForPublishDbInDefaultEnv = new IndexerFactory(indexingSettingsForPublishDbInDefaultEnv);
+            indexerPoolForPreviewDbInDefaultEnv = new IndexerPool(indexingSettingsForPreviewDbInDefaultEnv,
+                maxIdleTimeForIndexerInSecs, idleTimeCheckIntervalInSecs);
```

Comment:
Duplication of the same code. maybe use this method instead
    private IndexerPool initIndexerPool(long maxIdleTimeForIndexerInSecs, long idleTimeCheckIntervalInSecs, IndexingSettings indexingSettingsForSpecificEnv) {
            return new IndexerPool(indexingSettingsForSpecificEnv,
                    maxIdleTimeForIndexerInSecs, idleTimeCheckIntervalInSecs);
        }